packages/confirmation-modal/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -99,10 +104,15 @@ export const ConfirmationModal = React.forwardRef(
             disabled={!confirmEnabled || submitDisabled}
             onClick={onConfirm}
             className={buttonStyle}
+            data-testid="lg-confirmation-modal-footer-button"
```

Comment:
What about `lg-confirmation-modal-footer-confirm-button` since the button below has `cancel-button`

Reply:
Updated... what do you think?